### PR TITLE
Sanitise blog titles

### LINKED ISSUE #2: Blog posts title aren't sanitised
screen shot 2016-11-14 at 21 06 20

[Content is sanitised](https://github.com/zooniverse/Social/blob/442fd2bf2a0f8cabc5684ba7e0f1e73828494aa7/lib/zooniverse_social/posts.rb#L26) and so should be the title.

### PR DESCRIPTION
**Pull request by simoneduca**
Fixes #2

### CODE REVIEW

File: spec/zooniverse_social/posts_spec.rb [hunk 1]
```diff
@@ -28,14 +28,22 @@ module ZooniverseSocial
         expect(blog_updater).to have_received :update
         expect(daily_updater).to have_received :update
       end
+
+      it 'calls the daily updater' do
```
Comment (camallen) [inline spec/zooniverse_social/posts_spec.rb]: Isn't this testing the mock behaviour setup via allows in the before block?

File: spec/zooniverse_social/posts_spec.rb [hunk 2]
```diff
@@ -28,14 +28,22 @@ module ZooniverseSocial
...
+      end
     end
 
     describe '#update' do
       let(:blog_response) do
         {
           'posts' => [{
             'ID' => 123,
-            'title' => 'title1',
+            'title' => '<p>title1[&hellip;]</p>',
```
Comment (camallen) [inline spec/zooniverse_social/posts_spec.rb:39]: Are these values tested to be cleaned anywhere?

Comment (simoneduca) [reply spec/zooniverse_social/posts_spec.rb:39]: Here? https://github.com/zooniverse/Social/blob/sanitise-blog-titles/spec/zooniverse_social/posts_spec.rb#L114

File: spec/zooniverse_social/posts_spec.rb (hunk 1 above)
Comment (simoneduca) [reply spec/zooniverse_social/posts_spec.rb]: Yes, my bad I left that in but it is redundant sort of. Anyhow, I don't think we want the test to fail unless `:new` is called, so shall I just remove it?

**Review by camallen — APPROVED** [review]
LGTM

**Comment by simoneduca** [comment]
Cheers. It looks like for making this live I will have to get Talk to require it from git rather than as a gem. Or wait till Parrish is back.

**Comment by camallen** [comment]
@simoneduca - yeah seems parrish is the only one with rights on that gem, https://rubygems.org/gems/zooniverse_social

git ref will get it deployed via the build in the meantime.

**Comment by simoneduca** [comment]
Thanks, will ping Parrish later to ask him to publish a new version of the gem. Atm my fix doesn't really seem to work, but that's my problem.

**Pull request merged**